app/src/main/java/com/arygm/quickfix/ui/search/SearchWorkerProfileResult.kt

```diff
@@ -0,0 +1,150 @@
+package com.arygm.quickfix.ui.search
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.LocationOn
+import androidx.compose.material3.*
+import androidx.compose.material3.MaterialTheme.colorScheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.arygm.quickfix.R
+import com.arygm.quickfix.ui.elements.QuickFixButton
+import com.arygm.quickfix.ui.theme.poppinsFontFamily
+import com.arygm.quickfix.ui.theme.poppinsTypography
+
+@Composable
+fun SearchWorkerProfileResult(
+    profileImage: Int,
+    name: String,
+    category: String,
+    rating: Float,
+    reviewCount: Int,
+    location: String,
+    price: String,
+    onBookClick: () -> Unit
+) {
+  Card(
+      shape = RoundedCornerShape(8.dp),
+      modifier = Modifier.fillMaxWidth().padding(vertical = 10.dp, horizontal = 10.dp),
+      elevation = CardDefaults.cardElevation(10.dp),
+      colors = CardDefaults.cardColors().copy(containerColor = colorScheme.surface)) {
+        Row(
+            modifier = Modifier.fillMaxWidth().padding(8.dp),
+            verticalAlignment = Alignment.CenterVertically) {
+              Image(
+                  painter = painterResource(id = profileImage),
+                  contentDescription = null,
+                  modifier = Modifier.clip(RoundedCornerShape(8.dp)).size(100.dp).aspectRatio(1f),
+                  contentScale = ContentScale.FillBounds)
+
+              Spacer(modifier = Modifier.width(8.dp))
+
+              Column(
+                  modifier = Modifier.weight(0.6f).height(100.dp),
+                  verticalArrangement = Arrangement.SpaceAround,
+              ) {
+                Column(
+                    verticalArrangement = Arrangement.Top,
+                ) {
+                  Row(verticalAlignment = Alignment.CenterVertically) {
+                    Text(
```

Comment:
A lot of these text composables could be refactored into a single composable that you call accross the whole screen. Thus, avoiding a lot of duplication.